iroh-rpc-types/src/macros.rs

```diff
@@ -19,6 +19,11 @@ macro_rules! proxy_serve {
                         Ok(())
                     }
 
+                    #[cfg(feature = "grpc")]
+                    $crate::Addr::GrpcHttp2Lookup(name) => {
+                        anyhow::bail!("cannot serve on lookup address: {}", name);
```

Comment:
Aren't we moving away from anyhow?

Reply:
The conclusion was yes we are, but not yet right now - only sometime before 1.0.